src/spurt/workflows/emcf/_settings.py

```diff
@@ -12,9 +12,13 @@ class SolverSettings:
 
     Parameters
     ----------
-    worker_count: int
+    t_worker_count: int
         Number of workers for temporal unwrapping in parallel. Set value to <=0
         to let workflow use default workers (ncpus - 1).
+    s_worker_count: int
+        Number of workers for spatial unwrapping in parallel. Set value to <=0
+        to let workflow use (ncpus - 1).
+        Default = 1 (i.e. unwrap one ifg in space at a time).
```

Comment:
Let's use complete English sentences in docstring descriptions, please.

```suggestion
        Defaults to 1 (i.e. unwrap one interferogram in space at a time).
```